### DISCUSSION TITLE
How to get the Arctic Code Vault???

### DISCUSSION THREAD

**Discussion by weslleykelnner** [Schweinepriester/github-profile-achievements#43] [Q&A] [answered]
How to get the Arctic Code Vault badge?

**Comment by Schweinepriester** [comment]
You can't anymore:

> Earnable?
> ❌

> Earned by
> Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)

> The 02/02/2020 snapshot archived in the GitHub Arctic Code Vault swept up every active public GitHub repository. It included every repo with any commits between the announcement at GitHub Universe on November 13th and 02/02/2020; every repo with at least 1 star and any commits from the year before the snapshot; and all repos with at least 250 stars. (It also included gh-pages for those repositories.)

**Reply by weslleykelnner** [reply]
Thanks

**Comment by lincolnvc** [comment, marked as answer]
Is this the same question I asked? But, researching here I saw that it was a project that took place in 2020 that sent some open source projects to a repository in the arctic buried 250 meters away. This badge is no longer available.

**Reply by weslleykelnner** [reply]
tks
